README.md

```diff
@@ -3,10 +3,19 @@
 Bib Buddy offers a platform for concentrated literary work. For that, we create an open atmosphere for your thoughts and ideas during research. Our service makes your literary work as intuitive as never before!
 
 ## Functionality
-Bib Buddy is a digital notebook. It offers the possiblity to create notes for all literary works you work with and organize your literature in your personal library into different "shelves". Notes can be textual or audio/visual. Data about books is provided by German libraries.
+### Your library
+Bib Buddy offers the possibility to retrieve data for books by scanning their barcodes, typing their ISBNs or simply adding the data manually in a form. All books can be structured in shelves in your personal library inside the app. Naturally, that library is searchable.
+
+### Your thoughts
+To support with mobile literature work, we offer to save notes relating to a book either in textual form, with your usual formatting options (text style, alignment...) or in auditory form to quickly store a thought in your app.
+
+### Sharing
+Bib Buddy can share your data and notes about a book, a complete shelf or even your whole library as a bibTeX file. This file can be imported from Zotero or other literature software so you can make the best out of the thoughts you gathered within the app!
```

Comment:
```suggestion
With Bib Buddy you can share your data and notes about a book, a complete shelf or even your whole library as a bibTeX file. This file can be imported from Zotero or other literature software so you can make the best out of the thoughts you gathered within the app!
```
I suggest to phrase it like that since you are sharing the data through Bib Buddy - Bib Buddy does not do it by itself, that might be irritating since users might think this is happening permanently in the background. As well it is more precise.